### PR TITLE
Fixing EKS for new helm charts and updating product version to latest

### PR DESCRIPTION
**Pull request by vinayagg**
*Issue #, if available:*
none
*Description of changes:*
1. updated product version from 6.11.1 to 6.17.0
2. renamed files to make it easier to automate cicd jobs
3. updated certifiacte validation expressions as old expression was failing for us
4. changed verifyServerCertificate = false as verification fails with new version of java and mysql driver

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by cdenneen** [comment]
@vinayagg I'm curious if the ssl verification has to do with the RDS ca-2015 -> ca-2019 change that happened. Also could possibly be ca-certificates is out of date in the existing eks ami's which should allow for the ssl verification to be re-enabled. (Guessing the rename of artifactory to rt6 is because jfrog uses rt as the name for artifactory in their cli tool, probably not verify clear to most?)

**Comment by cdenneen** [comment]
@vinayagg 
- isn't the `SMLicensesName` still needed as a parameter that gets passed to the nested stacks?
- the `artifactoryConf` is no longer necessary to provide?

File: ci/jfrog-new-vpc-rt6-ec2.json [hunk 1]
```diff
@@ -55,10 +55,6 @@
         "ParameterKey": "CertificateDomain",
         "ParameterValue": "localdomain"
     },
-    {
```
Comment (cdenneen) [inline ci/jfrog-new-vpc-rt6-ec2.json]: @vinayagg Isn't this still needed?

File: templates/jfrog-rt6-eks-core-workload.template.yaml [hunk 2]
```diff
@@ -233,53 +233,6 @@ Resources:
               partition: artifactory-primary
             service:
               loadBalancerSourceRanges:  [ "${AccessCIDR}" ]
-            artifactoryConf: |
```
Comment (cdenneen) [inline templates/jfrog-rt6-eks-core-workload.template.yaml]: @vinayagg Isn't this still needed?

File: ci/jfrog-new-vpc-rt6-ec2.json (hunk 1 above)
Comment (vinayagg) [reply ci/jfrog-new-vpc-rt6-ec2.json]: This was duplicate field, I removed this one but on line 75, SMLicensesName still exists

File: templates/jfrog-rt6-eks-core-workload.template.yaml (hunk 2 above)
Comment (vinayagg) [reply templates/jfrog-rt6-eks-core-workload.template.yaml]: This was overriding the config from core product. however the values have been merged back into core product and we do not need to override. I have tested and these templates work fine without the override now (in fact they break with the override for some other reasons)

**Comment by vinayagg** [comment]
> @vinayagg I'm curious if the ssl verification has to do with the RDS ca-2015 -> ca-2019 change that happened. Also could possibly be ca-certificates is out of date in the existing eks ami's which should allow for the ssl verification to be re-enabled. (Guessing the rename of artifactory to rt6 is because jfrog uses rt as the name for artifactory in their cli tool, probably not verify clear to most?)

Because we are creating templates for 
- artifactory7
- jfrog container registry6
- jfrog container registry7
We needed a better way to name the templates. Hence the change.

**Pull request closed without merging**